### PR TITLE
Created an empty project.

### PR DESCRIPTION
**Pull request by UserProgrammer**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (franzvezuli) [inline .idea/compiler.xml:1]: We should add these to .gitignore too.

File: .idea/encodings.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (franzvezuli) [inline .idea/encodings.xml:1]: We should add these to .gitignore too.

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (franzvezuli) [inline .idea/misc.xml:1]: We should add these to .gitignore too.

File: .idea/vcs.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (franzvezuli) [inline .idea/vcs.xml:1]: We should add these to .gitignore too.

File: lindenmayer-system-api.iml [hunk 5]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (franzvezuli) [inline lindenmayer-system-api.iml]: We should add these to .gitignore too.

File: mvnw [hunk 6]
```diff
@@ -0,0 +1,286 @@
+#!/bin/sh
```
Comment (franzvezuli) [inline mvnw]: this can be removed I think

File: mvnw.cmd [hunk 7]
```diff
@@ -0,0 +1,161 @@
+@REM ----------------------------------------------------------------------------
```
Comment (franzvezuli) [inline mvnw.cmd]: This can be removed

File: src/main/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplication.kt [hunk 8]
```diff
@@ -0,0 +1,11 @@
+package com.condinglemurs.lindenmayersystemapi
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+
+@SpringBootApplication
+class LindenmayerSystemApiApplication
+
+fun main(args: Array<String>) {
+	runApplication<LindenmayerSystemApiApplication>(*args)
```
Comment (franzvezuli) [inline src/main/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplication.kt]: Should we agree on 2 spaces for this project?

File: src/main/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplication.kt [hunk 9]
```diff
@@ -0,0 +1,11 @@
+package com.condinglemurs.lindenmayersystemapi
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+
+@SpringBootApplication
+class LindenmayerSystemApiApplication
```
Comment (franzvezuli) [inline src/main/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplication.kt]: Seems redundant? Why not just `LindenmayerSystemApi`

File: src/main/resources/application.yml [hunk 10]
```diff
@@ -0,0 +1 @@
+
```
Comment (franzvezuli) [inline src/main/resources/application.yml]: Remove until we need to add something

File: src/test/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplicationTests.kt [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.junit4.SpringRunner
+
+@RunWith(SpringRunner::class)
+@SpringBootTest
+class LindenmayerSystemApiApplicationTests {
+
+	@Test
```
Comment (franzvezuli) [inline src/test/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplicationTests.kt]: Remove this class until we need it?

File: .idea/compiler.xml (hunk 1 above)
Comment (UserProgrammer) [reply .idea/compiler.xml:1]: Completed as discussed.

File: .idea/encodings.xml (hunk 2 above)
Comment (UserProgrammer) [reply .idea/encodings.xml:1]: Keeping as discussed.

File: .idea/misc.xml (hunk 3 above)
Comment (UserProgrammer) [reply .idea/misc.xml:1]: Keeping as discussed.

File: .idea/vcs.xml (hunk 4 above)
Comment (UserProgrammer) [reply .idea/vcs.xml:1]: Keeping as discussed.

File: lindenmayer-system-api.iml (hunk 5 above)
Comment (UserProgrammer) [reply lindenmayer-system-api.iml]: Keeping as discussed.

File: mvnw (hunk 6 above)
Comment (UserProgrammer) [reply mvnw]: Done.

Comment (UserProgrammer) [reply mvnw]: Done.

File: mvnw.cmd (hunk 7 above)
Comment (UserProgrammer) [reply mvnw.cmd]: Done.

File: src/main/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplication.kt (hunk 9 above)
Comment (UserProgrammer) [reply src/main/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplication.kt]: Done.

File: src/main/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplication.kt (hunk 8 above)
Comment (UserProgrammer) [reply src/main/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplication.kt]: Yes.

File: src/main/resources/application.yml (hunk 10 above)
Comment (UserProgrammer) [reply src/main/resources/application.yml]: Done.

File: src/test/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplicationTests.kt (hunk 11 above)
Comment (UserProgrammer) [reply src/test/kotlin/com/condinglemurs/lindenmayersystemapi/LindenmayerSystemApiApplicationTests.kt]: Done.

File: pom.xml [hunk 12]
```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
```
Comment (franzvezuli) [inline pom.xml]: change to spaces instead of tab

**Review by franzvezuli — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 12 above)
Comment (UserProgrammer) [reply pom.xml]: Done

**Review by franzvezuli — APPROVED** [review]

**Pull request merged**